routes/pullrequests.js

```diff
@@ -2,6 +2,37 @@ const express = require('express')
 const router = express.Router()
 const pullRequestController = require('../controllers/pullRequestsController')
 
+/**
+ * @swagger
+ * /pullrequests/stale:
+ *   get:
+ *     summary: All open Pull Requests in Real Dev Squad
+ *     tags:
+ *       - Pull Requests
+ *     responses:
+ *       200:
+ *         description: Pull Requests
+ *         content:
+ *           application/json:
+ *              schema:
+ *                type: object
+ *                properties:
+ *                  message:
+ *                    type: string
+ *                    example: Stale PRs
+ *                  pullRequests:
+ *                    type: array
+ *                    items:
+ *                      $ref: '#/components/schemas/pullRequests'
```

Comment:
This schema seems to be just slightly different from the `pullRequests` schema, like this one has got `username` of the author as well.

I'm not sure about this, but should we create a new schema just for one/two change(s)?

Reply:
You are absolutely right though @swarajpure! Great catch!

There is a BIG problem with the username, or rather usernames.
And I want @sumitd94 @whyDontI to resolve it.
We need to implement a Remote Dictionary Server (Redis) for a few reasons. We will have a session on it.